scripts/anonymizeDb.ts

```diff
@@ -0,0 +1,67 @@
+import config from '../utils/config';
+import knexModule from 'knex';
+import { psychologistsTable } from '../db/tables';
+import faker from 'faker';
+
+faker.locale = 'fr';
+
+const PROD_HOSTNAME = 'https://santepsy.etudiant.gouv.fr';
+
+const knexConfig = require('../knexfile');
+
+const knex = knexModule(knexConfig);
+
+const fakeFirstnames = (real: string) : string => {
+  const nbFirstnames = real ? real.split(' ').length - 1 : 0;
+
+  let fake = faker.name.firstName();
+  for (let i = 1; i < nbFirstnames; i++) {
+    fake += ` ${faker.name.middleName()}`;
+  }
+  return fake;
+};
+
+const fakeLastname = () : string => faker.name.lastName();
+
+const fakeEmail = (real: string) : string => (real ? faker.internet.exampleEmail() : real);
```

Comment:
i don't get this ?

Reply:
email is not mandatory (only personalEmail is), the idea is to keep them undefined, null or whatever the value is and replace it only if it has a value.